env.yml

```diff
@@ -0,0 +1,99 @@
+  - blas=1.0=mkl
+  - bottleneck=1.3.5=py310ha9d4c09_0
+  - brotli=1.0.9=h5eee18b_7
+  - brotli-bin=1.0.9=h5eee18b_7
+  - bzip2=1.0.8=h7b6447c_0
+  - ca-certificates=2022.10.11=h06a4308_0
+  - certifi=2022.12.7=py310h06a4308_0
+  - contourpy=1.0.5=py310hdb19cb5_0
+  - cycler=0.11.0=pyhd3eb1b0_0
+  - dbus=1.13.18=hb2f20db_0
+  - expat=2.4.9=h6a678d5_0
+  - fontconfig=2.14.1=h52c9d5c_1
+  - fonttools=4.25.0=pyhd3eb1b0_0
+  - freetype=2.12.1=h4a9f257_0
+  - giflib=5.2.1=h7b6447c_0
+  - glib=2.69.1=he621ea3_2
+  - gst-plugins-base=1.14.0=h8213a91_2
+  - gstreamer=1.14.0=h28cd5cc_2
+  - icu=58.2=he6710b0_3
+  - intel-openmp=2021.4.0=h06a4308_3561
+  - jpeg=9e=h7f8727e_0
+  - kiwisolver=1.4.2=py310h295c915_0
+  - krb5=1.19.2=hac12032_0
+  - lcms2=2.12=h3be6417_0
+  - ld_impl_linux-64=2.38=h1181459_1
+  - lerc=3.0=h295c915_0
+  - libbrotlicommon=1.0.9=h5eee18b_7
+  - libbrotlidec=1.0.9=h5eee18b_7
+  - libbrotlienc=1.0.9=h5eee18b_7
+  - libclang=10.0.1=default_hb85057a_2
+  - libdeflate=1.8=h7f8727e_5
+  - libedit=3.1.20221030=h5eee18b_0
+  - libevent=2.1.12=h8f2d780_0
+  - libffi=3.4.2=h6a678d5_6
+  - libgcc-ng=11.2.0=h1234567_1
+  - libgomp=11.2.0=h1234567_1
+  - libllvm10=10.0.1=hbcb73fb_5
+  - libpng=1.6.37=hbc83047_0
+  - libpq=12.9=h16c4e8d_3
+  - libstdcxx-ng=11.2.0=h1234567_1
+  - libtiff=4.4.0=hecacb30_2
+  - libuuid=1.41.5=h5eee18b_0
+  - libwebp=1.2.4=h11a3e52_0
+  - libwebp-base=1.2.4=h5eee18b_0
+  - libxcb=1.15=h7f8727e_0
+  - libxkbcommon=1.0.1=hfa300c1_0
+  - libxml2=2.9.14=h74e7548_0
+  - libxslt=1.1.35=h4e12654_0
+  - lz4-c=1.9.4=h6a678d5_0
+  - matplotlib=3.6.2=py310h06a4308_0
+  - matplotlib-base=3.6.2=py310h945d387_0
+  - mkl=2021.4.0=h06a4308_640
+  - mkl-service=2.4.0=py310h7f8727e_0
+  - mkl_fft=1.3.1=py310hd6ae3a3_0
+  - mkl_random=1.2.2=py310h00e6091_0
+  - munkres=1.1.4=py_0
+  - ncurses=6.3=h5eee18b_3
+  - nspr=4.33=h295c915_0
+  - nss=3.74=h0370c37_0
+  - numexpr=2.8.4=py310h8879344_0
+  - numpy=1.23.4=py310hd5efca6_0
+  - numpy-base=1.23.4=py310h8e6c178_0
+  - openssl=1.1.1s=h7f8727e_0
+  - packaging=22.0=py310h06a4308_0
+  - pandas=1.5.2=py310h1128e8f_0
+  - pcre=8.45=h295c915_0
+  - pillow=9.3.0=py310hace64e9_1
+  - pip=22.3.1=py310h06a4308_0
+  - ply=3.11=py310h06a4308_0
+  - pyparsing=3.0.9=py310h06a4308_0
+  - pyqt=5.15.7=py310h6a678d5_1
+  - python=3.10.8=h7a1cb2a_1
+  - python-dateutil=2.8.2=pyhd3eb1b0_0
+  - pytz=2022.7=py310h06a4308_0
+  - qt-main=5.15.2=h327a75a_7
+  - qt-webengine=5.15.9=hd2b0992_4
+  - qtwebkit=5.212=h4eab89a_4
+  - readline=8.2=h5eee18b_0
+  - setuptools=65.5.0=py310h06a4308_0
+  - sip=6.6.2=py310h6a678d5_0
+  - six=1.16.0=pyhd3eb1b0_1
+  - sqlite=3.40.0=h5082296_0
+  - tk=8.6.12=h1ccaba5_0
+  - toml=0.10.2=pyhd3eb1b0_0
+  - tornado=6.2=py310h5eee18b_0
+  - tzdata=2022g=h04d1e81_0
+  - wheel=0.37.1=pyhd3eb1b0_0
+  - xz=5.2.8=h5eee18b_0
+  - zlib=1.2.13=h5eee18b_0
+  - zstd=1.5.2=ha4553b6_0
+  - pip:
+      - pyqt5-sip==12.11.0
+prefix: /home/rishabh/ENTER/envs/mle-dev
```

Comment:
Remove the prefix line. This is because we want the environment to be deployable anywhere.

Reply:
ok